### PR TITLE
fix(client): nav control focus (Triggered with next page/animation at the same time)

### PR DESCRIPTION
**Pull request by FyisFe**
Hi Anthony,

I noticed that you have fixed this issue. However, the same problem also occurs if my mouse does not leave the navigation bar. 

I think your solution is already perfect, just provide another solution for you to check. 😄 

Furthermore, I find that the total page number on the navigation bar is always 1 less than the actual one, and I fix it by adding 1. 😄

### CODE REVIEW

**Comment by antfu** [comment]
Thanks for looking into, but actually we could have a better solution for that. 

Just adding 'BUTTON' and 'A' to the list should work
https://github.com/slidevjs/slidev/blob/d9314e72aa6b01be7ea2a4387b396c7ba9e94222/packages/client/state/index.ts#L18-L18

**Comment by FyisFe** [comment]
Have done that. Thanks so much for your reply.

File: packages/client/internals/MenuButton.vue [hunk 1]
```diff
@@ -22,7 +22,7 @@ onClickOutside(el, () => {
 
 <template>
   <div ref="el" class="flex relative">
-    <div :class="{ disabled }" @click="value = !value">
+    <div :class="{ disabled }" @click="value = !value" @keyup.space.prevent>
```
Comment (antfu) [inline packages/client/internals/MenuButton.vue:25]: Could convert this to `<button/>` I think

File: packages/client/internals/NavControls.vue [hunk 2]
```diff
@@ -7,19 +7,15 @@ import { configs } from '../env'
...
     default: 'fixed',
   },
 })
-
 const md = breakpoints.smaller('md')
 const { isFullscreen, toggle: toggleFullscreen } = fullscreen
-
 const presenterLink = computed(() => `${location.origin}/presenter/${currentPage.value}`)
 const nonPresenterLink = computed(() => `${location.origin}/${currentPage.value}`)
-
```
Comment (antfu) [inline packages/client/internals/NavControls.vue:22]: Can you bring them back? Thanks

Comment (FyisFe) [reply packages/client/internals/NavControls.vue:22]: Sure. Sorry for not noticing them 😅

**Pull request merged**